app/assets/javascripts/school_administrator_dashboard/discipline_dashboard/SchoolDisciplineDashboard.js

```diff
@@ -187,7 +204,7 @@ export default class SchoolDisciplineDashboard extends React.Component {
   }
 
   studentTableRows(students) {
-    const filteredStudents = this.filterStudents(students);
+    const filteredStudents = this.filterStudents(students, true);
```

Comment:
If you revise the argument to be `options = {}` this can help make it more obvious what's happening at the call site.  Right now, reading this line of code, there's no indication of what `true` means.

Using the `options = {}` argument let's you write code like this at the call site:
```
this.filterStudents(students, { shouldFilterSelectedCategory: true })
```

and express explicitly what you want that function to be doing.  I'd recommend this as a good rule of thumb in general.

Reply:
Thanks for this! I was sure there was a better syntax I was missing.